mirage/yaml/yaml_generator.py

```diff
@@ -379,6 +389,13 @@ def create_inputs(self):
             file_dict['primary_dither_num'] = np.int(primary_dither)
             subpix_dither = (tot_dith-1) % subpixtot
             file_dict['subpix_dither_num'] = subpix_dither + 1
+            file_dict['subarray_def_file'] = self.global_subarray_definition_files[self.info['Instrument'][i].lower()]
+            file_dict['readpatt_def_file'] = self.global_readout_pattern_files[self.info['Instrument'][i].lower()]
+            file_dict['crosstalk_file'] = self.global_crosstalk_files[self.info['Instrument'][i].lower()]
+            file_dict['filtpupilcombo_file'] = self.global_filtpupilcombo_files[self.info['Instrument'][i].lower()]
+            file_dict['flux_cal_file'] = self.global_flux_cal_files[self.info['Instrument'][i].lower()]
+            file_dict['psfpath'] = self.global_psfpath[self.info['Instrument'][i].lower()]
+
             fname = self.write_yaml(file_dict)
             yamls.append(fname)
 
```

Comment:
Further down, at line 408, the `mosaic_numbers` are being defined based on the old file naming scheme. Related to a previous comment, we should figure out what the correct naming scheme is and then make sure this matches. Currently it does not, so `yaml_generator` is telling me about Observation 000, 100, 200, etc. instead of 001, 002...

Reply:
Can we take this to a separate PR?

Reply:
Definitely.